src/librustdoc/html/render/print_item.rs

```diff
@@ -600,7 +596,6 @@ fn item_trait(w: &mut Buffer, cx: &Context<'_>, it: &clean::Item, t: &clean::Tra
         for t in types {
             trait_item(w, cx, t, it);
         }
-        write_loading_content(w, "</div>");
```

Comment:
The `</div>` itself still need to be there, doesn't it?

Reply:
Indeed, good catch!